styx-scheduler-service/src/main/java/com/spotify/styx/Scheduler.java

```diff
@@ -185,6 +185,11 @@ private void tick0() {
             .sorted(comparingLong(i -> i.runState().timestamp()))
             .collect(toList());
 
+    final String message = String.format("Instances: active=%d, eligible=%d, timedOut=%d",
+        activeStates.size(), eligibleInstances.size(), timedOutInstances.size());
+    LOG.info(message);
+    tracer.getCurrentSpan().addAnnotation(message);
```

Comment:
Shall we verify this invocation in test case?

Reply:
It's not possible to mock `tracer.getCurrentSpan()` 😫

Reply:
nvm, i thought that `tracer` was injected.